boards/variscite/imx8mp_var_dart/imx8mp_var_dart_mimx8ml8_m7.yaml

```diff
@@ -0,0 +1,17 @@
+#
+# Copyright 2025 Variscite Ltd.
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+identifier: imx8mp_var_dart/mimx8ml8/m7
+name: Variscite DART-MX8M-PLUS M7
+type: mcu
+arch: arm
+ram: 128
+flash: 128
+toolchain:
+  - zephyr
+  - gnuarmemb
+supported:
+  - uart
```

Comment:
You can probably add a few more of the available/enabled peripherals to get better coverage in CI